internal/service/alertconfiguration/model_alert_configuration_test.go

```diff
@@ -0,0 +1,537 @@
+package alertconfiguration_test
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/conversion"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/service/alertconfiguration"
+	"go.mongodb.org/atlas-sdk/v20231115002/admin"
+)
+
+const (
+	group               string  = "GROUP"
+	previousIntervalMin int     = 10
+	intervalMin         int     = 5
+	delayMin            int     = 0
+	enabled             bool    = true
+	disabled            bool    = false
+	previousOperator    string  = "MORE_THAN"
+	operator            string  = "LESS_THAN"
+	threshold           float64 = 99.0
+	units               string  = "RAW"
+	mode                string  = "AVERAGE"
+)
+
+func TestNotificationSDKToTFModel(t *testing.T) {
+	testCases := []struct {
+		name                      string
+		SDKResp                   *[]admin.AlertsNotificationRootForGroup
+		currentStateNotifications []alertconfiguration.TfNotificationModel
+		expectedTFModel           []alertconfiguration.TfNotificationModel
+	}{
+		{
+			name: "Complete SDK response",
```

Comment:
we could include an additional test case for notification type that requires a sensitive attribute, such as pager duty.

Reply:
great, thanks for adding that case